components/button/stories/button.stories.js

```diff
@@ -210,6 +222,12 @@ const PendingButton = ({
 			${CustomButton({
 				...args,
 			})}
+			${when(window.isChromatic(), () =>
```

Comment:
Could we add Chromatic only wrapping text examples to all of these? Or add the progress bar to the wrapping text story for Chromatic? I tested it and the positioning looks good (centered both directions); it would be helpful to represent in VRTs.

Reply:
Good call, just pushed this addition